.github/workflows/ci.yml

```diff
@@ -149,3 +154,27 @@ jobs:
 
       - name: Cargo nextest
         run: cargo nextest run --workspace
+
+ test_wasm:
```

Comment:
super pedantic; needs an extra space in front (I can see only 1). Also how about `tests_wasm` (added "s") to be consistent with the other tests_x steps?